extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -251,6 +256,18 @@ require([
 				handleClick(event, category);
 			}
 		});
+
+		// Track clicks in the To-do List module
+		$('.community-page-todo-list-module-edit').on('mousedown touchstart', 'a', function (event) {
+			handleClick(event, 'community-page-todo-list-module');
+		});
+
+		$('.community-page-todo-list-module-content').on('mousedown touchstart', 'a', function (event) {
+			track({
+				category: 'community-page-todo-list-module',
+				label: event.delegateTarget.getAttribute('data-tracking'),
```

Comment:
Yeah, the AC specifically say to use the same label for all content inside the .community-page-todo-list-module-content div, so I will drop the data-tracking attribute for that one and just hardcode the data to send.